crates/js-component-bindgen/src/transpile_bindgen.rs

```diff
@@ -0,0 +1,711 @@
+use crate::files::Files;
```

Comment:
Transpile bindgen is included, and made a feature. Raw bindgen is removed. Instead componentize bindgen can be built on top of function bindgen directly. The entry point to the bindgen is treated as a function instead of a stateful options initialization.